src/gesture-recognizers.js

```diff
@@ -762,18 +760,16 @@ export class PinchRecognizer extends GestureRecognizer {
     if (touches && touches.length == 1) {
       return true;
     }
-    // If already in the middle of a pinch event, ignore additional touches.
-    if (this.eventing_ && touches && touches.length > 2) {
-      return true;
-    }
     else if (touches && touches.length >= 2) {
```

Comment:
Better to just turn this into an if-statement since the `if` is an early return.